FordLabs/retroquest
# Feedback dialog redesign

## Overview
Redesigns the feedback dialog to support the new UI direction. Also adds way better mobile support than what was in the old version.

**Desktop**
![image](https://user-images.githubusercontent.com/6293337/43937713-55523f92-9c2d-11e8-9fa7-0e07ebc1f84d.png)

**Mobile** (IPhone SE / 5)
![image](https://user-images.githubusercontent.com/6293337/43937720-63b9c5aa-9c2d-11e8-8fe8-0e2eb92c2fde.png)

## Testing Instructions
1. Submit feedback like normal, using the "Feedback" button in the header
2. See if the feedback can be retrieved either from the endpoint or the database.
